tests/unit/test_prometheus_metrics.py

```
@@ -1,621 +1,422 @@
 
-Tests the Prometheus exposition format generator for schema v1.
-"""
-
-import os
 import re
-import sys
-import tempfile
 import time
-import unittest
+from pathlib import Path
+from typing import Tuple
```

Ruff (0.15.7)

[warning] `typing.Tuple` is deprecated, use `tuple` instead

(UP035)